library/core/src/num/f128.rs

```diff
@@ -829,8 +828,7 @@ impl f128 {
     /// ```
     #[inline]
     #[unstable(feature = "f128", issue = "116909")]
-    // #[unstable(feature = "num_midpoint", issue = "110840")]
-    pub fn midpoint(self, other: f128) -> f128 {
+    pub const fn midpoint(self, other: f128) -> f128 {
```

Comment:
You added a `const` here, you will also have to add a `rustc_const_unstable`.
```suggestion
    #[rustc_const_unstable(feature = "f128", issue = "116909")]
    pub const fn midpoint(self, other: f128) -> f128 {
```
Same for the f16 function.